maven-plugins/stager-maven-plugin/src/test/java/io/helidon/build/maven/stager/StagingTaskTest.java

```diff
@@ -96,4 +87,66 @@ public String describe(Path dir, Map<String, String> variables) {
             return "test";
         }
     }
+
+    private static final class ContextTestImpl implements StagingContext {
+
+        @Override
+        public void unpack(Path archive, Path target, String excludes, String includes) {
+
```

Comment:
We should throw an UnsupportedOperationException in this methods (unpack, archive, resolve*, createTempDirectory)